comtypes/server/register.py

```diff
@@ -310,21 +310,21 @@ def _registry_entries(self, cls):
         ):
             exe = sys.executable
             if " " in exe:
-                exe = '"%s"' % exe
+                exe = f'"{exe}"'
             if not hasattr(sys, "frozen"):
                 if not __debug__:
-                    exe = "%s -O" % exe
+                    exe = f"{exe} -O"
                 script = os.path.abspath(sys.modules[cls.__module__].__file__)
                 if " " in script:
-                    script = '"%s"' % script
+                    script = f'"{script}"'
                 append(
                     HKCR,
                     "CLSID\\%s\\LocalServer32" % reg_clsid,
                     "",
-                    "%s %s" % (exe, script),
+                    f"{exe} {script}",
                 )
             else:
-                append(HKCR, "CLSID\\%s\\LocalServer32" % reg_clsid, "", "%s" % exe)
+                append(HKCR, "CLSID\\%s\\LocalServer32" % reg_clsid, "", f"{exe}")
```

Comment:
If you're concerned about the `{}` in f-strings being escaped by a backslash(`\`), there's no need to worry.
As shown below, `f"\\{foo}\\"` would not be escaped. 

```py
>>> from comtypes import GUID
>>> reg_clsid = GUID()
>>> f"CLSID\\{reg_clsid}\\LocalServer32"
'CLSID\\{00000000-0000-0000-0000-000000000000}\\LocalServer32'
>>> "CLSID\\%s\\LocalServer32" % reg_clsid
'CLSID\\{00000000-0000-0000-0000-000000000000}\\LocalServer32'
>>> "CLSID\\%s\\LocalServer32" % reg_clsid == f"CLSID\\{reg_clsid}\\LocalServer32"
True
```

Thus, it seems this could be rewritten as follows. Additionally, there might be other similar parts in the code that can be replaced, so please check them as well. 


```diff
-                append(HKCR, "CLSID\\%s\\LocalServer32" % reg_clsid, "", f"{exe}")
+                append(HKCR, f"CLSID\\{reg_clsid}\\LocalServer32", "", f"{exe}")
```

That said, I've only tested this on Python 3.11, so if it doesn't work with older versions of Python, leaving it as is should be fine.